common/base/include/claragenomics/logging/logging.hpp

```diff
@@ -93,7 +93,10 @@
 #include <spdlog/spdlog.h>
 #endif
 
-namespace claragenomics
+namespace claraparabricks
+{
+
+namespace genomeworks
 {
 namespace logging
 {
```

Comment:
Do we want to have a new line between namespaces? We did not do it previously.
I'm fine with both, we should just pick one and stick to it

Reply:
It's not consistent right now. Some files have a new line and some don't. I want to make it consistent with a new line. I'll push a change to fix the older ones too

Reply:
Updated